common/tools/field_test_log_plotter/ftl_plotter.py

```diff
@@ -1101,10 +1119,13 @@ def plot_rx_throughput_and_mcs_per_time(self, _mac):
 
         # Set throughput label names
         rx_throughput_label = 'RX throughput [bit/s]'
```

Comment:
Should here be used similar LABEL_RX_THROUGHPUT definition (or whatever called in python as LABEL_THROUGHPUT below? It could look nicer.